packages/cycle-scripts-one-fits-all/template/tsconfig.json

```diff
@@ -2,7 +2,7 @@
     "compileOnSave": false,
     "compilerOptions":
     {
-        "module": "ES6",
```

Comment:
I raised this for the config intellisense https://github.com/Microsoft/vscode/issues/25416